pkg/apis/componentconfig/types.go

```diff
@@ -150,6 +150,9 @@ type KubeletConfiguration struct {
 	// default /var/run/kubernetes). If tlsCertFile and tlsPrivateKeyFile
 	// are provided, this flag will be ignored.
 	CertDirectory string `json:"certDirectory"`
+	// BootstrapKubeConfig is the path to config file or directory of files that
+	// will be used to get client certificate for kubelet.
+	BootstrapKubeconfig string `json:"bootstrapKubeconfig"`
```

Comment:
Why? @vishh and I are pushing for `KubeletConfiguration` to have parity with flags so it can also be used in on-disk config as well. /cc @vishh.